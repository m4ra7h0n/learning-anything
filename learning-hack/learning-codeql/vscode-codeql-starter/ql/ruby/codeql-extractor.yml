name: "ruby"
display_name: "Ruby"
version: 0.1.0
column_kind: "utf8"
legacy_qltest_extraction: true
github_api_languages:
  - Ruby
scc_languages:
  - Ruby
file_types:
  - name: ruby
    display_name: Ruby files
    extensions:
       - .rb
  - name: erb
    display_name: Ruby templates
    extensions:
       - .erb
