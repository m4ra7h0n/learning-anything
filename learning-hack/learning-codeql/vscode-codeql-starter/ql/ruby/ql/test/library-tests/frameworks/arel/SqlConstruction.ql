import codeql.ruby.Concepts
import codeql.ruby.DataFlow

query predicate sqlConstructions(SqlConstruction c, DataFlow::Node sql) { sql = c.getSql() }
