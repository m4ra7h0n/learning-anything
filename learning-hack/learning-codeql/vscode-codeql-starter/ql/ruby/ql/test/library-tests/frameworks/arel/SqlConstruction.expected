| arel.rb:3:8:3:18 | call to sql | arel.rb:3:17:3:17 | x |
| arel.rb:8:8:8:18 | call to sql | arel.rb:8:17:8:17 | x |
